helm/cilium/README.md

```diff
@@ -1,6 +1,6 @@
 # cilium
 
-![Version: 0.25.1](https://img.shields.io/badge/Version-0.25.1-informational?style=flat-square) ![AppVersion: 1.15.6](https://img.shields.io/badge/AppVersion-1.15.6-informational?style=flat-square)
+![Version: 0.25.1](https://img.shields.io/badge/Version-0.25.1-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
```

Comment:
Not something to be fixed in this PR. IMO we should make sure this is being run in our Release PRs so that these badge links are updated. Or we remove them from the Readme template.

Reply:
Oh yes, this will cause the same problem we've seen in the envoy-gateway repo during the release. I'll remove the badge.